docs/plugins/filters/geoip.asciidoc

```diff
@@ -44,7 +44,7 @@ A `[geoip][location]` field is created if
 the GeoIP lookup returns a latitude and longitude. The field is stored in
 http://geojson.org/geojson-spec.html[GeoJSON] format. Additionally,
 the default Elasticsearch template provided with the
-{logstash-ref}/plugins-outputs-elasticsearch.html[elasticsearch output] maps
+<<plugins-outputs-elasticsearch,`elasticsearch` output>> maps
```

Comment:
I think we need to use the logstash-ref style urls here?

Reply:
@jsvd  Good catch! You are correct. I'll fix it here and then fix it in the source.